client/app/cda/cda.controller.js

```diff
@@ -1,9 +1,7 @@
 angular.module("managerApp")
-  .controller("CdaCtrl", function ($scope, $state, CdaService) {
-      "use strict";
-      $scope.$on("$stateChangeSuccess", function () {
-          if ($state.includes("paas.cda")) {
-              CdaService.initDetails($state.params.serviceName);
-          }
-      });
-  });
\ No newline at end of file
+    .controller("CdaCtrl", ($transitions, CdaService) => {
+        $transitions.onSuccess({ to: "paas.cda.**" }, transition => {
```

Comment:
@frenautvh I've used the $transitions filter instead managing scopes.

Reply:
you're right!

Reply:
@Netnix i think a hook is created each time controller is instantiated right ? if it's the case you might still need to remove hook when scope is destroyed or the callback might be called more than once